types/jslib-html5-camera-photo/index.d.ts

```diff
@@ -16,9 +16,9 @@ interface ImageTypes {
 type FacingMode = 'user' | 'environment';
 type ImageType = 'png' | 'jpg';
 
-interface Resolution {
-    height?: number;
-    width?: number;
+interface PartialMediaTrackConstraints {
```

Comment:
I'm not sure if the name is the best one (for consuming this types, it's supposed to help). As first time user, for me it's just resolution, as per public docs:
https://github.com/MABelanger/jslib-html5-camera-photo#start-the-default-mode-facing-mode--resolution
or via api screening:
https://github.com/MABelanger/jslib-html5-camera-photo/blob/4a776199313e0a31bbbdcece3c49a96c979052e8/src/lib/index.js#L141
Neither 'partial' (which is a detail) nor 'track', whihc is relation to other type via index lookup.
I'm about to start camera using facial mode and resolution concepts

Reply:
Yeah, I understand what you're saying. I named it like this because the `idealResolution` parameter is really `Partial<MediaTrackConstraints>` with some fields omitted.

Is this the change you would recommend?

```ts
interface Resolution {
    height?: MediaTrackConstraints['height'];
    width?: MediaTrackConstraints['width'];
}
```